database/backend/architecture.json

```diff
@@ -1,23 +1,30 @@
 [
-    {
-        "name": "Microsoft Architecture Center",
-        "description": "Best practices and patterns for building applications on Microsoft Azure. Also provides suggestions for architecting solutions on Azure by using established patterns and practices.",
-        "url": "https://learn.microsoft.com/en-us/azure/architecture/",
-        "category": "backend",
-        "subcategory": "architecture"
-    },
-    {
-        "name": "AWS Architecture Center",
-        "description": "Get Started Architecting on AWS. It provides reference architecture diagrams, vetted architecture solutions, Well-Architected best practices, patterns, icons, and more.",
-        "url": "https://aws.amazon.com/architecture/",
-        "category": "backend",
-        "subcategory": "architecture"
-    },
-    {
-        "name": "Google Cloud Architecture Center",
-        "description": "Discover reference architectures, guidance, and best practices for building or migrating your workloads on Google Cloud.",
-        "url": "https://cloud.google.com/architecture/",
-        "category": "backend",
-        "subcategory": "architecture"
-    }
-]
\ No newline at end of file
+  {
+    "name": "Microsoft Architecture Center",
+    "description": "Best practices and patterns for building applications on Microsoft Azure. Also provides suggestions for architecting solutions on Azure by using established patterns and practices.",
+    "url": "https://learn.microsoft.com/en-us/azure/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  },
+  {
+    "name": "AWS Architecture Center",
+    "description": "Get Started Architecting on AWS. It provides reference architecture diagrams, vetted architecture solutions, Well-Architected best practices, patterns, icons, and more.",
+    "url": "https://aws.amazon.com/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  },
+  {
+    "name": "Google Cloud Architecture Center",
+    "description": "Discover reference architectures, guidance, and best practices for building or migrating your workloads on Google Cloud.",
+    "url": "https://cloud.google.com/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  },
+  {
+    "name": "Google Cloud Architecture Center",
+    "description": "Discover reference architectures, guidance, and best practices for building or migrating your workloads on Google Cloud.",
+    "url": "https://cloud.google.com/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  }
```

Comment:
Is this a duplicate @Divyansh-0? If not, I highly recommend adding detail to the description to make them more distinctive.